### PR TITLE
Sign up form

### PR DESCRIPTION
**Pull request by ktylke**
Add sign up form with segmented control button

### CODE REVIEW

File: TOZ_iOS/SignUpViewController.swift [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    @IBOutlet weak var phoneNumberInput: TextInputView!
+
+    @IBOutlet weak var goalSegmentedControl: UISegmentedControl!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureView()
+    }
+
+    override func didReceiveMemoryWarning() {
```
Comment (ArkadiuszBanas) [inline TOZ_iOS/SignUpViewController.swift]: We don't need that.

File: TOZ_iOS/SignUpViewController.swift [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+
+//    @IBAction func cancelButtonTapped(_ sender: UIButton) {
```
Comment (ArkadiuszBanas) [inline TOZ_iOS/SignUpViewController.swift]: Please remove that if it's not needed.

File: TOZ_iOS/SignUpViewController.swift [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+        firstNameInput.placeholder = " Imię..."
+        surnameInput.placeholder = " Nazwisko..."
+        emailInput.placeholder = " E-mail..."
+        phoneNumberInput.placeholder = " Numer telefonu..."
+    }
+    /*
+     // MARK: - Navigation
+
+     // In a storyboard-based application, you will often want to do a little preparation before navigation
```
Comment (ArkadiuszBanas) [inline TOZ_iOS/SignUpViewController.swift]: Same here.

**Comment by piotr-tobolski** [comment]
There are some conflicts in the storyboard that aren't trivial so please solve them

**Pull request merged**